apps/eip-7702-gas-sponsorship/foundry.toml

```diff
@@ -0,0 +1,9 @@
+[profile.default]
+src = "src"
+out = "out"
+libs = ["lib"]
+evm_version = "prague"
+no_cache = true
+via_ir = true
+
```

Comment:
➜  eip-7702-gas-sponsorship git:(eip-7702-gas-sponsorship-guide) ✗ forge build 
[⠃] Compiling...2025-06-03T23:35:06.472388Z ERROR foundry_compilers::resolver: failed to resolve versions
[⠊] Compiling...
Error: Encountered invalid solc version in src/SimpleDelegate.sol: No solc version exists that matches the version requirement: ^0.8.29
Encountered invalid solc version in src/SimpleDelegatePart2.sol: No solc version exists that matches the version requirement: ^0.8.29
Encountered invalid solc version in src/SimpleDelegate.sol: No solc version exists that matches the version requirement: ^0.8.29
Encountered invalid solc version in src/SimpleDelegatePart2.sol: No solc version exists that matches the version requirement: ^0.8.29

can we set the version here